AmmoDriver.md

```diff
@@ -241,6 +244,7 @@ Any entity with an `ammo-body` component can also have 1 or more `ammo-shape` co
 - **Mesh** (`mesh`) – Creates a 1:1 concave collision shape with the triangles of the meshes of the entity. May only be used on `static` bodies. This is the least performant shape, however they can work very well for static environments if the following is observed:
   - Avoid using meshes with very high triangle density relative to size of convex objects (primitives and hulls) colliding with the mesh. E.g. avoid meshes where an object could collide with dozens or more triangles in a single spot.
   - Avoid very high poly meshes in general and use mesh decimation (simplification) if possible.
+- **Heightfield** (`heightfield`) – Similar to a mesh shape, but you must provide an array of heights and the distance between those values. E.g. `heightfieldData: [[0, 0, 0], [0, 1, 0], [0, 0, 0]]` and `heightfieldDistance: 1` will create a 3x3 meter heightfield with a height of 0 except for the center with a height of 1.
```

Comment:
Probably minor, but how large would you expect the heightfield data to be in practice? I'm wondering if a flat array (9x1 in this case) would be preferable so you can use a typed array with `setAttribute` for larger areas.

Reply:
They can get pretty big, but not so big that I think you'd benefit much from that sort of optimization. An upside to keeping this an array of arrays is that the dimensions of the heightfield can be inferred based on the size of the arrays, but I suppose you could add additional properties to explicitly state what those are. Above all else though, I think as it is now makes it easier to "read" for debugging or manual creation.